dusty/commands/setup.py

```diff
@@ -26,12 +28,35 @@ def _get_default_specs_repo():
     _pretty_print_key_info(constants.CONFIG_SPECS_REPO_KEY)
     return _get_raw_input('Input the full name of your specs repo, e.g. github.com/gamechanger/example-dusty-specs: ')
 
+def _get_contents_of_file(file_location):
+    with open(file_location) as f:
+        return f.readlines()
+
+def _append_to_file(file_location, new_line):
+    with open(file_location, 'a') as f:
+        f.write(new_line)
+
+def _setup_nginx_config(nginx_conf_location):
+    include_path = '{}/servers'.format(nginx_conf_location)
+    if not isdir(include_path):
+        mkdir(include_path)
+    _append_to_file('{}/nginx.conf'.format(nginx_conf_location), 'include servers/*;')
+    return include_path
+
 def _get_nginx_includes_dir():
-    _pretty_print_key_info(constants.CONFIG_NGINX_DIR_KEY)
-    default_nginx_config_value = get_config_value(constants.CONFIG_NGINX_DIR_KEY)
-    if _get_raw_input('Does your nginx config look for extra configs in the default location of {}? (y/n): '.format(default_nginx_config_value)).upper() == 'Y':
-        return default_nginx_config_value
-    return _get_raw_input('Input the path where your nginx config pulls extra configs: ')
+    for nginx_conf_location in constants.NGINX_CONFIG_FILE_LOCATIONS:
+        file_location = '{}/nginx.conf'.format(nginx_conf_location)
+        if isfile(file_location):
+            contents = _get_contents_of_file(file_location)
+            for line in contents:
+                if line.startswith('include'):
+                    include_folder = line.replace('include ', '').replace('/*;', '').replace('\n', '').strip()
+                    return '{}/{}'.format(nginx_conf_location, include_folder)
+            if _get_raw_input('\n'.join(textwrap.wrap('You have non standard nginx config setup. Can we add "include servers/*;" to the end of your nginx config file ({})? If you select no, your nginx forwarding will not work. (y/n) '.format(file_location), 80))).upper() == 'Y':
+                return _setup_nginx_config(nginx_conf_location)
+            else:
+                return ''
+    return _get_raw_input('\n'.join(textwrap.wrap('You have non standard nginx config setup. Could not find an nginx.conf file. Please input the full path of the directory your nginx config includes. ', 80)))
```

Comment:
This is really jank if anyone makes it this far, wonder if there's a better way we could handle this